test/utils/init.ts

```diff
@@ -13,6 +15,12 @@ const Client = new Dynamo({
             accessKeyId: 'test',
             secretAccessKey: 'test',
         }),
+        logger: {
```

Comment:
I'm not familiar with this param to the DynamoDBClient. What is dynamoExecutedCommandsTracer doing for you?

Reply:
This is used to track all the commands performed on dynamodb, if you see the tests on `loader.ts` I check the following 

```js
 expect(dynamoExecutedCommandsTracer).toHaveBeenCalledWith(
      expect.objectContaining({
        commandName: 'BatchGetItemCommand'
      })
    );
    ```